src/renderer/ipc.ts

```diff
@@ -13,7 +13,7 @@ function getImplementation(): IPCInterface {
 
   logger.log('IPC was not configured in preload script, falling back to custom protocol and fetch');
 
-  fetch(`${PROTOCOL_SCHEME}://${IPCChannel.PING}/sentry_key`).catch(() =>
+  fetch(`${PROTOCOL_SCHEME}://${IPCChannel.PING}/sentry_key`, { method: 'POST', body: '' }).catch(() =>
```

Comment:
Just a thought: Alternatively we could filter for this request in `beforeBreadcrumb` but I think this is also fine.

Reply:
I would love integrations to [have more hooks](https://github.com/getsentry/sentry-javascript/issues/4790#issuecomment-1357818411) like for `beforeBreadcrumb`!